src/content/snapshot/delay.mdx

```diff
@@ -99,26 +98,27 @@ If you need additional control when Chromatic captures a snapshot, you can adjus
   // Adjust this import to match your framework (e.g., nextjs, vue3-vite)
   import type { Meta, StoryObj } from "@storybook/your-framework";
 
-  /*
-  * Replace the storybook/test import with `@storybook/test` and adjust the stories accordingly if you're not using Storybook 9.0.
-  * Refer to the Storybook documentation for the correct package and imports for earlier versions.
-  */
+/\*
+
+- Replace the storybook/test import with `@storybook/test` and adjust the stories accordingly if you're not using Storybook 9.0.
+- Refer to the Storybook documentation for the correct package and imports for earlier versions.
```

Comment:
@winkerVSbecks There is one small item here. When you can, could you check the formatting on this page? It seems that it has been lost, and it renders incorrectly.